lib/rucio/core/heartbeat.py

```diff
@@ -58,7 +58,7 @@ def _sanity_check(executable, hostname, hash_executable=None, session=None):
     """
     if executable:
         if not hash_executable:
-            hash_executable = hashlib.sha256(executable).hexdigest()
+            hash_executable = hash(executable)
```

Comment:
doesn't this nameclash with the built-in hash() function?

Reply:
didn't knew that there is a builtin function. Seems to override it. Should I change anyway?

Reply:
maybe a slight rename, yes. just for consistency sake.